src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -99,16 +99,19 @@ private function addRoute(RouteCollection $routeCollection, string $resourceClas
             throw new RuntimeException('Either a "route_name" or a "method" operation attribute must exist.');
         }
 
-        if (isset($operation['controller'])) {
-            $controller = $operation['controller'];
-        } else {
-            $actionName = sprintf('%s_%s', strtolower($operation['method']), $collection ? 'collection' : 'item');
-            $controller = self::DEFAULT_ACTION_PATTERN.$actionName;
-        }
+        $collectionString = $collection ? 'collection' : 'item';
+
+        $actionName = sprintf('%s_%s', $operationName, $collectionString);
+        $controller = $operation['controller'] ?? self::DEFAULT_ACTION_PATTERN.sprintf('%s_%s', strtolower($operation['method']), $collectionString);
```

Comment:
As long as `$operationName` matches `strtolower($operation['method'])`, it should be fine.